### ISSUE TITLE
Error: Attempted to load trait "AttributeValueField"...

### ISSUE THREAD

**Issue by crabnky** [msgphp/symfony-demo-app#32]
Hi again :)

I just run composer update and now getting this:
```

Attempted to load trait "AttributeValueField" from namespace "MsgPhp\Eav\Entity\Fields".
Did you forget a "use" statement for another namespace?
--
in UserAttributeValue.php (line 17)
```
My composer:
```
"msgphp/domain": "@dev",
"msgphp/user": "@dev",
"msgphp/user-bundle": "dev-master",
```
It's just a mistake, or something has changed and I have to make changes to my code?

**Comment by ro0NL** [comment]
`msgphp/eav: @dev` i guess :) im working on a new release.

**Comment by crabnky** [comment]
Uh, sorry, my bad. :(
Have to pay attention to configuration as now I can see:
`The path "doctrine.orm.resolve_target_entities.MsgPhp\User\Entity\Username" cannot contain an empty value, but got null.` - now I'm in the middle of other work, so will get back to this later.
Thanks!

**Comment by DevMDamien** [comment]
Seems missing this in your config:
```
'username_lookup' => [
    ['target' => UserEmail::class, 'field' => 'email'],
],
```
[see the config file](https://github.com/msgphp/symfony-demo-app/blob/e2b68f9da3abbcaef1d2be5ef90b5802c272aea1/config/packages/msgphp.php#L24-L26)

**Comment by ro0NL** [comment]
tagged 0.2.4, should be good now :)

**Comment by ro0NL** [comment]
@crabnky thanks for all your issues :) truly valuable!

**Comment by ro0NL** [comment]
@crabnky actually solved in 0.2.7 :} There was indeed an issue with ` trait "AttributeValueField"`
